### PR TITLE
Fix typo: "/" -> "\"

### PR DESCRIPTION
**Pull request by exigow**

### CODE REVIEW

File: TESTING.md
Comment (mkondratek) [inline TESTING.md]: pls mention macOS btw: `<kbd>option</kbd> + <kbd>\</kbd>`

File: TESTING.md
Comment (mkondratek) [inline TESTING.md]: pls mention macOS btw: `<kbd>option</kbd> + <kbd>\</kbd>`

File: TESTING.md
Comment (mkondratek) [inline TESTING.md]: pls mention macOS btw: `<kbd>option</kbd> + <kbd>\</kbd>`

File: TESTING.md
Comment (exigow) [reply TESTING.md]: done

File: TESTING.md
Comment (exigow) [reply TESTING.md]: done

**Comment by exigow** [comment]
Fixed all threads. Waiting for approval.

**Review by mkondratek — APPROVED** [review]

**Pull request merged**
